core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -869,7 +869,7 @@ func (ec *EthConfirmer) handleInProgressAttempt(ctx context.Context, lggr logger
 			"blockHeight", blockHeight,
 			"id", "RPCTxFeeCapExceeded",
 		)
-		return ec.orm.DeleteInProgressAttempt(ctx, attempt)
+		return ec.orm.DeleteInProgressAttempt(attempt, pg.WithParentCtx(ctx))
```

Comment:
IMHO this is the wrong direction. We should prefer to use standard contexts, especially in exported APIs.

Reply:
I understand where you're coming from. The main benefits of having the variadic  any -> qopts is that you can pass in any db specific arguments. However this does come with the downside of needing explicit typecasting. 

How would you recommend passing in the Queryer then to these Generic ORM methods? The Queryer is specific to the postgres db and is needed to pass in a sqlx.Transaction between methods to keep all SQL statements within the same transaction. Without the variadic any -> qopts conversion we'd have to pass it in as is which would pigeon-hole the ORM to be dealing with the sqlx-supported databases

Reply:
Agree with Jordan, makes more sense to me at this point considering generic ORM to revert to the standard context approach.